content/technology/solutions/altinn-studio/designer/functional/build-app/define-process/_index.md

```diff
@@ -12,7 +12,7 @@ In Altinn Studio the developer will need to define the process for the app.
 
 In current version of Altinn Studio the only support for process definition is to upload a BPMN process file and the only task type we support are FormFilling
 
-[See Process Architecture for more details how process is defined](https://docs.altinn.studio/architecture/application/altinn-apps/app/app-backend/process/)
+[See Process Architecture for more details how process is defined](/architecture/application/altinn-apps/app/app-backend/process/)
```

Comment:
Oh my, these are very old... Should be cleaned up, sorry. Use this link instead: 

`/technology/architecture/capabilities/runtime/processing/process/`